src/main/java/meteordevelopment/meteorclient/gui/WidgetScreen.java

```diff
@@ -318,10 +320,7 @@ public void removed() {
             Input.setCursorStyle(CursorStyle.Default);
 
             loopWidgets(root, widget -> {
-                if (widget instanceof WTextBox textBox) {
```

Comment:
While the cleanup is nice, please keep this to changes actually relevant to 1.20